src/app/CreateRecording/CustomRecordingForm.tsx

```diff
@@ -243,6 +263,65 @@ export const CustomRecordingForm = (props) => {
           </SplitItem>
         </Split>
       </FormGroup>
+      <ExpandableSection toggleTextExpanded="Hide advanced options" toggleTextCollapsed="Show advanced options">
+        <Form>
+          <FormGroup label="Maximum age" fieldId="maxAge">
+            <Split hasGutter={true}>
+              <SplitItem isFilled>
+                <TextInput
+                  value={maxAge}
+                  isRequired
+                  type="number"
+                  id="maxAgeDuration"
+                  aria-describedby="maxAgeDuration-helper"
```

Comment:
I don't recall off the top of my head what needs to be added for these `describedby` accessibility attributes, but there should be a text label or something with the id `maxAgeDuration-helper` I think that this "links" to? Same for all the other `aria-describedby` attributes on other elements.

Reply:
Also I'm sure these are missing pretty much everywhere else right now...

Reply:
I changed the aria-describedby to aria-labels because it makes more sense. I think that other parts of the code have aria-describedby that don't link anywhere, so Im not sure If I should change them as well?

Reply:
Nice. That sounds like a great enhancement, but certainly as a separate Issue/PR.